pages/linux/pacstrap.md

```diff
@@ -1,7 +1,7 @@
 # pacstrap
 
 > Install packages using Pacman to the specified new root directory.
-> More information: <https://man.archlinux.org/man/pacstrap.8>.
+> More information: <https://manned.org/pacstrap>.
```

Comment:
```suggestion
> More information: <https://man.archlinux.org/man/pacstrap.8>.
```
`pacman` and all its tools are Arch specific.

Reply:
Oh yeah, this one slipped my filters :) I'll remove.

Reply:
[Done](https://github.com/tldr-pages/tldr/pull/5790/commits/c4c78403adc536382658fcde58d679c7cf17d160).